nav2_system_tests/src/system/CMakeLists.txt

```diff
@@ -73,8 +61,7 @@ ament_add_test(test_dynamic_obstacle
   TIMEOUT 180
   ENV
     TEST_DIR=${CMAKE_CURRENT_SOURCE_DIR}
-    TEST_MAP=${PROJECT_SOURCE_DIR}/maps/map_circular.yaml
-    TEST_WORLD=${PROJECT_SOURCE_DIR}/worlds/turtlebot3_ros2_demo_obstacle.world
+    TEST_WORLD=${PROJECT_SOURCE_DIR}/worlds/no_roof_small_warehouse_with_obstacle.world
     GAZEBO_MODEL_PATH=${PROJECT_SOURCE_DIR}/models
```

Comment:
Is `GAZEBO_MODEL_PATH` still needed in CMakeLists.txt?

Reply:
Done.